diversity_analyzer.py

```diff
@@ -27,7 +27,7 @@
 import html_report_writer
 
 test_reads = os.path.join(home_directory, "test_dataset/merged_reads.fastq")
-test_dir = os.path.join(home_directory, "divan_test")
+test_dir = "divan_test"
```

Comment:
Does that corrects the following issue?
https://ablab.myjetbrains.com/youtrack/issue/IG-60